### PR TITLE
[icons] feat: new icon "buggy"

### PR DESCRIPTION
**Pull request by cameronjoyner**
Created a new vehicle icon - a buggy!! Excited to share it with the world!

### CODE REVIEW

**Comment by palantirtech** [comment]
Thanks for your interest in [palantir/blueprint](https://github.com/palantir/blueprint), @cameronjoyner! Before we can accept your pull request, you need to sign our contributor license agreement - just visit https://cla.palantir.com/ and follow the instructions. Once you sign, I'll automatically update this pull request.

File: resources/icons/16px/buggy.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
...
..."evenodd" d="M13.9285 0.628648C13.7615 0.21119 13.3364 -0.0434973 12.8895 0.00615448L3.88955 1.00615C3.57936 1.04062 3.30306 1.21793 3.14249 1.48554L0.430971 6.00473C0.18751 6.03837 3.05176e-05 6.24729 3.05176e-05 6.5V9.5C3.05176e-05 9.77615 0.223888 10 0.500031 10H3.29847C3.63908 10 3.96954 10.1159 4.23551 10.3287L6.18768 11.8904C6.27634 11.9614 6.38649 12 6.50003 12H10C10.1574 12 10.3056 11.9259 10.4 11.8L11.3 10.6C11.5833 10.2223 12.0279 10 12.5 10H15.5C15.7762 10 16 9.77615 16 9.5V5.5C16 5.34717 15.9301 5.20272 15.8103 5.10789C15.7739 5.07915 15.7342 5.05592 15.6924 5.03848L13.9285 0.628648ZM12.3605 2.094L12.3541 2.07796L9.18829 2.42971L10.3094 5.51264L12.3605 2.094ZM12.827 3.2602L11.1785 6.00765L13.6887 5.41436L12.827 3.2602ZM9.60292 6.49613L8.16555 2.54335L4.60266 2.93923L2.76619 6H3.50003C3.63931 6 3.77228 6.0581 3.86691 6.1603L6.21846 8.69998H8.13241L9.58398 6.52263C9.59002 6.51357 9.59634 6.50473 9.60292 6.49613ZM2.5 16C3.88071 16 5 14.8807 5 13.5C5 12.1193 3.88071 11 2.5 11C1.11929 11 0 12.1193 0 13.5C0 14.8807 1.11929 16 2.5 16ZM13.5 16C14.8807 16 16 14.8807 16 13.5C16 12.1193 14.8807 11 13.5 11C12.1193 11 11 12.1193 11 13.5C11 14.8807 12.1193 16 13.5 16Z" fill="black"/>
```
Comment (aycai) [inline resources/icons/16px/buggy.svg]: I'd drop the `fill="black"`.

File: resources/icons/20px/buggy.svg [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
...9.3952 6.51116C19.5429 6.47953 19.6969 6.5163 19.8143 6.61123C19.9318 6.70616 20 6.84907 20 7.00007V11.5001C20 11.7762 19.7761 12.0001 19.5 12.0001H15.5704C15.0688 12.0001 14.6005 12.2507 14.3223 12.668L12.916 14.7774C12.8233 14.9165 12.6672 15.0001 12.5 15.0001H8C7.86739 15.0001 7.74021 14.9474 7.64645 14.8536L5.23223 12.4394C4.95093 12.1581 4.5694 12.0001 4.17157 12.0001H0.5C0.223858 12.0001 0 11.7762 0 11.5001V8.50007C0 8.22392 0.223858 8.00007 0.5 8.00007H1.32296L3.07152 3.62867C3.20128 3.30426 3.49096 3.0711 3.8356 3.01366L15.8356 1.01366ZM16.125 4.48611L17.356 6.94814L14.5985 7.53903L16.125 4.48611ZM14.5001 3.26384L12.9403 6.38328L12.6878 5.74556L11.8627 3.7034L14.5001 3.26384ZM4.71948 4.89394L9.84173 4.04023L10.8296 6.48512L11.7287 8.75569L10.2324 11.0001H7.70711L4.85355 8.14651C4.75979 8.05274 4.63261 8.00007 4.5 8.00007H3.47703L4.71948 4.89394ZM3 19.0001C4.65685 19.0001 6 17.6569 6 16.0001C6 14.3432 4.65685 13.0001 3 13.0001C1.34315 13.0001 0 14.3432 0 16.0001C0 17.6569 1.34315 19.0001 3 19.0001ZM17 19.0001C18.6569 19.0001 20 17.6569 20 16.0001C20 14.3432 18.6569 13.0001 17 13.0001C15.3431 13.0001 14 14.3432 14 16.0001C14 17.6569 15.3431 19.0001 17 19.0001Z" fill="black"/>
```
Comment (aycai) [inline resources/icons/20px/buggy.svg]: Same thing, `fill="black"`isn't needed.

**Comment by aycai** [comment]
Designs are solid, left a few comments on the svgs!

File: resources/icons/16px/buggy.svg (hunk 1 above)
Comment (adidahiya) [reply resources/icons/16px/buggy.svg]:
```suggestion
<path fill-rule="evenodd" clip-rule="evenodd" d="M13.9285 0.628648C13.7615 0.21119 13.3364 -0.0434973 12.8895 0.00615448L3.88955 1.00615C3.57936 1.04062 3.30306 1.21793 3.14249 1.48554L0.430971 6.00473C0.18751 6.03837 3.05176e-05 6.24729 3.05176e-05 6.5V9.5C3.05176e-05 9.77615 0.223888 10 0.500031 10H3.29847C3.63908 10 3.96954 10.1159 4.23551 10.3287L6.18768 11.8904C6.27634 11.9614 6.38649 12 6.50003 12H10C10.1574 12 10.3056 11.9259 10.4 11.8L11.3 10.6C11.5833 10.2223 12.0279 10 12.5 10H15.5C15.7762 10 16 9.77615 16 9.5V5.5C16 5.34717 15.9301 5.20272 15.8103 5.10789C15.7739 5.07915 15.7342 5.05592 15.6924 5.03848L13.9285 0.628648ZM12.3605 2.094L12.3541 2.07796L9.18829 2.42971L10.3094 5.51264L12.3605 2.094ZM12.827 3.2602L11.1785 6.00765L13.6887 5.41436L12.827 3.2602ZM9.60292 6.49613L8.16555 2.54335L4.60266 2.93923L2.76619 6H3.50003C3.63931 6 3.77228 6.0581 3.86691 6.1603L6.21846 8.69998H8.13241L9.58398 6.52263C9.59002 6.51357 9.59634 6.50473 9.60292 6.49613ZM2.5 16C3.88071 16 5 14.8807 5 13.5C5 12.1193 3.88071 11 2.5 11C1.11929 11 0 12.1193 0 13.5C0 14.8807 1.11929 16 2.5 16ZM13.5 16C14.8807 16 16 14.8807 16 13.5C16 12.1193 14.8807 11 13.5 11C12.1193 11 11 12.1193 11 13.5C11 14.8807 12.1193 16 13.5 16Z"/>
```

File: resources/icons/20px/buggy.svg (hunk 2 above)
Comment (adidahiya) [reply resources/icons/20px/buggy.svg]:
```suggestion
<path fill-rule="evenodd" clip-rule="evenodd" d="M15.8356 1.01366C16.2686 0.941491 16.6981 1.16019 16.8944 1.55284L19.3757 6.51535L19.3952 6.51116C19.5429 6.47953 19.6969 6.5163 19.8143 6.61123C19.9318 6.70616 20 6.84907 20 7.00007V11.5001C20 11.7762 19.7761 12.0001 19.5 12.0001H15.5704C15.0688 12.0001 14.6005 12.2507 14.3223 12.668L12.916 14.7774C12.8233 14.9165 12.6672 15.0001 12.5 15.0001H8C7.86739 15.0001 7.74021 14.9474 7.64645 14.8536L5.23223 12.4394C4.95093 12.1581 4.5694 12.0001 4.17157 12.0001H0.5C0.223858 12.0001 0 11.7762 0 11.5001V8.50007C0 8.22392 0.223858 8.00007 0.5 8.00007H1.32296L3.07152 3.62867C3.20128 3.30426 3.49096 3.0711 3.8356 3.01366L15.8356 1.01366ZM16.125 4.48611L17.356 6.94814L14.5985 7.53903L16.125 4.48611ZM14.5001 3.26384L12.9403 6.38328L12.6878 5.74556L11.8627 3.7034L14.5001 3.26384ZM4.71948 4.89394L9.84173 4.04023L10.8296 6.48512L11.7287 8.75569L10.2324 11.0001H7.70711L4.85355 8.14651C4.75979 8.05274 4.63261 8.00007 4.5 8.00007H3.47703L4.71948 4.89394ZM3 19.0001C4.65685 19.0001 6 17.6569 6 16.0001C6 14.3432 4.65685 13.0001 3 13.0001C1.34315 13.0001 0 14.3432 0 16.0001C0 17.6569 1.34315 19.0001 3 19.0001ZM17 19.0001C18.6569 19.0001 20 17.6569 20 16.0001C20 14.3432 18.6569 13.0001 17 13.0001C15.3431 13.0001 14 14.3432 14 16.0001C14 17.6569 15.3431 19.0001 17 19.0001Z"/>
```

**Review by adidahiya — CHANGES_REQUESTED** [review]

**Review by adidahiya — APPROVED** [review]

**Pull request merged**